spec/liquidation.md

```diff
@@ -1 +0,0 @@
-# Liquidation
```

Comment:
We don't want to add the algorithm for the liquidation bot? Or is this covered in the margin section

Reply:
The liquidation bot will go through the margin contract. Will revert this and add a sentence on liquidation.